com.microsoft.mrtk.spatialmanipulation/Tests/Runtime/BoundsControlTests.cs

```diff
@@ -303,5 +303,107 @@ public IEnumerator TestNoHandlesToggleWhenMovingWithObjectManipulator([ValueSour
             yield return null;
             Assert.IsTrue(bc == null);
         }
+
+        [UnityTest]
+        public IEnumerator TestManipulationCursor([ValueSource(nameof(BoundsVisualsPrefabs))] string visualsPath)
```

Comment:
nit.... 

Scott had mentioned that the MRTK3 tests normally end with the word "Test"...so I guess rename `TestManipulationCursor` to `ManipulationCursorTest`

Reply:
In this file BoundsControlTests they usually start with Test